tests/test_storages.py

```diff
@@ -71,6 +73,29 @@ def test_json_readwrite(tmpdir):
     assert get('A short one') is None
 
 
+def test_create_dirs():
+    temp_dir = tempfile.gettempdir()
+    db_dir = ''
+    db_file = ''
+
+    while True:
```

Comment:
I don't feel good about this loop. Couldn't this make tests hang for some edge cases? I'd propose using pytest's [tmpdir](http://doc.pytest.org/en/latest/tmpdir.html) that's also used in other tests (e.g. [test_storages.py#L49](https://github.com/msiemens/tinydb/blob/873a82e0b0ed71e6f2dcd3332e26960e95622675/tests/test_storages.py#L49)).